SPECS/keyutils/keyutils.spec

```diff
@@ -1,13 +1,20 @@
 Summary:        Linux Key Management Utilities
 Name:           keyutils
-Version:        1.5.10
-Release:        5%{?dist}
+Version:        1.6.1
+Release:        1%{?dist}
 License:        GPLv2+ AND LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Base
 URL:            https://people.redhat.com/~dhowells/keyutils/
-Source0:        https://people.redhat.com/~dhowells/keyutils/keyutils-%{version}.tar.bz2
+Source0:        https://people.redhat.com/~dhowells/keyutils/%{name}-%{version}.tar.bz2
+%if %{with_check}
```

Comment:
Is there a reason to conditionalize the `Patch0` tag here? In the same way that we avoid conditionalizing sources/patches based on build architecture, I think we should avoid doing the same with RPM defines. I think it would be better to conditionalize the patch application in the `%prep` section, if conditionalization is needed at all.

Reply:
Good catch. It is not necessary and I forgot that this will cause us to build different SRPMs depending on the build options. I'll change this in a minute.